gctscript/modules/ta/indicators/indicators.go

```diff
@@ -0,0 +1,40 @@
+package indicators
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/gctscript/modules"
+)
+
+var supportedDurations = []string{"1m", "3m", "5m", "15m", "30m", "1h", "2h", "4h", "6h", "12h", "24h", "1d", "3d", "1w"}
```

Comment:
If you don't want to I will just resolve.

Reply:
happy with 1 month do we consider 1 month to be 28/29/30/31 days but?
maybe 4w instead?

Reply:
I don't think it would be a big deal because the candles returned will be dependant on the exchange and the time reference. 28 days (4 weeks) or 30 days seem fine.

Reply:
One month in the charting world usually starts from the 1st of every month, useful for our own OHLC generation but other 1month data would be exchange dependent

Reply:
Yeah fair enough

I originally just added everything that was included in https://github.com/thrasher-corp/gocryptotrader/blob/ce11dfcf39a6e84c5704cdffbca595978cb2809d/exchanges/kline/types.go#L11-L25 just to keep it inline

Reply:
This also is most likely not the best place for it to sit the method and the listed of supported durations is only use in OHLCV wrapper calls i will move it over to exchange module at some point today